internal/test/test.go

```diff
@@ -59,7 +59,7 @@ func MockHTTPServer(requestChan chan *Request) *mockServer {
 // ReqChan makes a channel for reading received requests one by one.
 // Use it in conjunction with MockHTTPServer
 func ReqChan() chan *Request {
-	return make(chan *Request, 1)
+	return make(chan *Request, 999)
```

Comment:
@lyoshenka any downsides to this? it was blocking for me when tested code was calling fake server more times than expected

Reply:
i don't really like the way we use channels here, even though im the one who wrote this. bumping the buffer to 999 just hides the problem, and i don't like hiding problems. i don't have a good solution yet, so this is prolly fine for now.